### PR TITLE
Shader example in FlxBunnyMark

### PR DESCRIPTION
**Pull request by Beeblerox**

### CODE REVIEW

**Comment by Gama11** [comment]
Very nice! I think I'm gonna look for a few more cool shaders to add...

File: Performance/FlxBunnyMark/source/PlayState.hx [hunk 1]
```diff
@@ -17,10 +18,11 @@ import flixel.util.FlxColor;
  */
 class PlayState extends FlxState
 {
-	public static inline var INITIAL_AMOUNT:Int = 1000;
+	public static inline var INITIAL_AMOUNT:Int = 100;
```
Comment (Gama11) [inline Performance/FlxBunnyMark/source/PlayState.hx]: Did you decrease this amount as a result of the shader changes / are they performance-intensive for you? 100 seems too little for a performance benchmark demo to me. I think this also impacts the amount of bunnies added with each button click.

File: Performance/FlxBunnyMark/source/PlayState.hx [hunk 2]
```diff
@@ -35,7 +37,17 @@ class PlayState extends FlxState
 	private var _offScreenButton:FlxButton;
 	private var _bunnyCounter:FlxText;
 	private var _fpsCounter:FlxText;
-
+	
+	#if !flash
```
Comment (Gama11) [inline Performance/FlxBunnyMark/source/PlayState.hx]: Would be nicer to have a `shaders_supported` haxedef that we define in the `Project.xml`. That way only one place has to be changed if the targets on which we support shaders change.

File: Performance/FlxBunnyMark/source/PlayState.hx [hunk 3]
```diff
@@ -120,6 +137,10 @@ class PlayState extends FlxState
 	{
 		super.update(elapsed);
 		
+		#if !flash
+		floodFill.uFloodFillY = 0.5 * (1.0 + Math.sin(Lib.getTimer() / 1000));
```
Comment (Gama11) [inline Performance/FlxBunnyMark/source/PlayState.hx]: Should use `FlxG.game.ticks` which caches the `Lib.getTimer()` value.

File: Performance/FlxBunnyMark/Project.xml [hunk 4]
```diff
@@ -33,6 +33,7 @@
 	
 	<!--------------------------------LIBRARIES------------------------------------->
 
+	<set name="next"/>
```
Comment (Gama11) [inline Performance/FlxBunnyMark/Project.xml]: I'm hesistant to change the default to `next` before we have a workaround for #228. We could probably just let the background be rendered without `FlxStrip` for OpenFL Next? A `FlxBackdrop` or something?

**Comment by Gama11** [comment]
Have you tried this on HTML5 (webgl needs to be enabled)? I think in theory it should work, but FlxBunnyMark bugs out for me with WebGL enabled even without these changes..

File: Performance/FlxBunnyMark/source/PlayState.hx (hunk 1 above)
Comment (Beeblerox) [inline Performance/FlxBunnyMark/source/PlayState.hx]: oops, don't know how it was changed, will change it back to default

**Pull request merged**